build.rs

```diff
@@ -1,21 +0,0 @@
-use std::process::Command;
```

Comment:
Hi @moubctez, cc @devendran-m

Sorry I am missing some info here, was there a decision to remove that build code ? because I just opened a [PR](https://github.com/casper-ecosystem/casper-client-rs/pull/175) to condition that git versioning.

this `GIT_SHA_SHORT` is used  in main.rs and I don't see any change regarding this in this PR, I am missing something ? thanks

Reply:
It must have been killed in action of conflict resolution. I've restored `build.rs`.

Reply:
Hi, Sorry but why are you reverting to vergen ?

This file has evolved I believe vergen was removed 

https://github.com/casper-ecosystem/casper-client-rs/blob/dev/build.rs

from https://github.com/casper-ecosystem/casper-client-rs/pull/149